### PR TITLE
add verify leader status

### PR DESCRIPTION
**Pull request by upccup**

### CODE REVIEW

File: src/manager/manager.go [hunk 1]
```diff
@@ -410,6 +418,7 @@ func (manager *Manager) SendAgentInitData(agent types.Node) {
 	taskEvents := manager.framework.Scheduler.HealthyTaskEvents()
 
 	for _, taskEvent := range taskEvents {
+		fmt.Printf("taskEvent:   %+v \n", taskEvent)
```
Comment (cmingxu) [inline src/manager/manager.go]: debug statment

File: src/manager/manager.go [hunk 2]
```diff
@@ -268,6 +268,10 @@ func (manager *Manager) LoadNodeData() error {
 }
 
 func (manager *Manager) AddNode(node types.Node) error {
```
Comment (cmingxu) [inline src/manager/manager.go]: Q:  all AddNode/RemoveNode requests are supposed to proxy to leader, is that right? Then why bother guard ?

File: src/manager/manager.go [hunk 3]
```diff
@@ -444,3 +452,11 @@ func (manager *Manager) SendAgentInitData(agent types.Node) {
 		logrus.Errorf("marshal janitor init data got error: %s", err.Error())
 	}
 }
+
+func (manager *Manager) LeaderAffirm() error {
```
Comment (cmingxu) [inline src/manager/manager.go]: LeaderAffirm? What is Affirm?

File: src/manager/manager.go (hunk 2 above)
Comment (upccup) [inline src/manager/manager.go]: yep, the add/remove node will proxy to leader

**Pull request merged**